anyway/request_params.py

```diff
@@ -87,10 +90,19 @@ def get_request_params_from_request_values(vals: dict) -> Optional[RequestParams
         start_time=start_time,
         end_time=end_time,
         lang=lang,
+        description=description
     )
     logging.debug(f"Ending get_request_params. params: {request_params}")
     return request_params
 
+def get_description_from_request_values(vals: dict) -> str:
+    news_flash_id = vals.get("news_flash_id")
+    if news_flash_id is None:
+        return None
+    news_flash = extract_news_flash_obj(news_flash_id)
```

Comment:
Hi Shay, I think it is worthwhile to avoid reading the news flash from database twice in `get_request_params_from_request_values()`.

Reply:
Done but please have a look at the changes made